packages/ui/src/views/repo/components/branch-info-bar.tsx

```diff
@@ -7,23 +7,21 @@ interface BranchInfoBarProps {
   defaultBranchName?: string
   spaceId: string
   repoId: string
-  currentBranch: {
-    behindAhead: {
-      ahead: number
-      behind: number
-    }
+  currentBranchDivergence: {
+    ahead: number
+    behind: number
   }
 }
 
 export const BranchInfoBar: FC<BranchInfoBarProps> = ({
   defaultBranchName = 'main',
   spaceId,
   repoId,
-  currentBranch
+  currentBranchDivergence
 }) => {
-  const { behindAhead } = currentBranch
-  const hasBehind = behindAhead.behind > 0
-  const hasAhead = behindAhead.ahead > 0
+  const { behind, ahead } = currentBranchDivergence
+  const hasBehind = behind && behind > 0
+  const hasAhead = ahead && ahead > 0
```

Comment:
Not an issue, but looks like a good case for checking for truthyness instead of existence and being over 0.
```suggestion
  const hasBehind = !!behind
  const hasAhead = !!ahead
```